src/super_gradients/training/losses/ohem_ce_loss.py

```diff
@@ -52,3 +57,40 @@ def forward(self, logits, labels):
         else:
             loss = loss[:num_mining]
         return torch.mean(loss)
+
+
+class OhemCELoss(OhemLoss):
+    """
+    OhemLoss - Online Hard Example Mining Cross Entropy Loss
+    """
+
+    def __init__(self,
+                 threshold: float,
+                 mining_percent: float = 0.1,
+                 ignore_lb: int = -100,
+                 num_pixels_exclude_ignored: bool = True):
+        ignore_lb = -100 if ignore_lb is None or ignore_lb < 0 else ignore_lb
+        criteria = nn.CrossEntropyLoss(ignore_index=ignore_lb, reduction='none')
+        super(OhemCELoss, self).__init__(threshold=threshold,
+                                         mining_percent=mining_percent,
+                                         ignore_lb=ignore_lb,
+                                         num_pixels_exclude_ignored=num_pixels_exclude_ignored,
+                                         criteria=criteria)
+
+
+class OhemBCELoss(OhemLoss):
+    """
+    OhemBCELoss - Online Hard Example Mining Binary Cross Entropy Loss
+    """
+
+    def __init__(self,
+                 threshold: float,
+                 mining_percent: float = 0.1,
+                 num_pixels_exclude_ignored: bool = True, ):
+        super(OhemBCELoss, self).__init__(threshold=threshold,
+                                          mining_percent=mining_percent,
+                                          num_pixels_exclude_ignored=num_pixels_exclude_ignored,
+                                          criteria=nn.BCEWithLogitsLoss(reduction='none'))
+
+    def forward(self, logits, labels):
+        return super(OhemBCELoss, self).forward(logits.squeeze(1), labels.float())
```

Comment:
This line will failed if num_classes > 1

Reply:
fixed.